CONTRIBUTING.md

```diff
@@ -11,15 +11,15 @@ make setup
 > This installs uv as a [standalone application][uv-install]. <br>
 > For more details, see also the [uv documentation][uv-docs]. <br>
 
-2. Initialize project dependencies with uv and install `pre-commit` hooks:
+2. Initialize project dependencies with `uv` and install `pre-commit` hooks:
 
 ```bash
 make install-dev
 make pre-commit-install
 ```
 
 This will install project dependencies into the currently active environment. If you would like to
-use uv's default behavior of managing a project-scoped environment, use uv commands directly to
+use uv's default behavior of managing a project-scoped environment, use `uv` commands directly to
```

Comment:
still missed one 😅

Reply:
I don't think it's helpful to use backticks when you're not describing a specific command being run, see the uv readme and the Poetry Readme https://github.com/astral-sh/uv?tab=readme-ov-file#installation

I'd prefer we not be super strict about this to lower the barrier to making docs contributions.

Reply:
Seems like a bad formatting on their end for that case. 
In the same document, any other tool (`pip`, `pip-tools`, `pipx`, `poetry`, etc.) is referenced using backticks.

Reply:
poetry doesn't use backticks for themselves on their own readme: https://github.com/python-poetry/poetry/blob/main/README.md I think it's valid to refer to a project without backticks, it's a proper noun that is meant to identify not refer to code

Reply:
It can either be a link to the project or backticks when referring to the commands, but it shouldn't simply be an unformatted word, because that assumes users know about it, which is not necessarily the case. Using either a link/backtick puts the emphasis about it referring to a specific tool/command.